Assets/code_src/TimedActionManager.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
```

Comment:
`TimedActionManager` keeps track of several `TimedActions` and triggers them as required.